scipy/misc/__init__.py

```diff
@@ -32,13 +28,25 @@
    imshow - Simple showing of an image through an external viewer [requires Pillow]
    info - Get help information for a function, class, or module
    lena - Get classic image processing example image Lena
+   toimage - Takes a numpy array and returns a PIL image [requires Pillow]
+   source - Print function source code
```

Comment:
Huh, it's just a GH way of showing the diff. I did not touch these ones. Do you want me to deprecate them as well in this PR?